doc/templates/verify_student/incourse_reverify.html

```diff
@@ -0,0 +1,35 @@
+<%page expression_filter="h"/>
```

Comment:
@shimulch, @aayushagra, can we avoid storing copies of the upstream template files in this repository? This will prevent issues that may arise when a newer Open edX release updates this template and things may not work as expected.

Reply:
@lgp171188, @aayushagra, I've removed the templates and added only what's needs to be changed on those templates.